coursebuilder/modules/explorer/_static/components/course-explorer/course-explorer.html

```diff
@@ -158,7 +175,23 @@ <h3 class="explorer-title">Featured Courses</h3>
         coursesLoaded: {
           type: Boolean,
           value: false,
-        }
+        },
+        registeredCourses: {
+          type: Array,
+          value: [],
+        },
+        dashboardCourses: {
```

Comment:
@andela-angene What's the difference between this and registeredCourses?

Reply:
It should be named registeredCourseIds since it contains just the ids of registered courses, I will change this.